ci-operator/step-registry/openshift-qe/installer/bm/day2/lvms/openshift-qe-installer-bm-day2-lvms-commands.sh

```
@@ -0,0 +1,170 @@
+  ssh ${SSH_ARGS} root@${bastion} '
+    for i in $(oc get node --no-headers -l node-role.kubernetes.io/worker --output custom-columns="NAME:.status.addresses[0].address"); do
+      for j in {0..7}; do
+        ssh -t '"${SSH_ARGS}"' core@$i sudo sgdisk --zap-all /dev/nvme${j}n1 2>/dev/null
+        ssh -t '"${SSH_ARGS}"' core@$i sudo wipefs -a /dev/nvme${j}n1 2>/dev/null
+      done
+    done'
+}
+
+wipe_sata() {
+  local bastion=$1
+  ssh ${SSH_ARGS} root@${bastion} '
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)